upgrade/1.3/Stage_0_Prerequisites.md

```diff
@@ -190,14 +190,14 @@ If the following command does not complete successfully, check if the `TOKEN` en
 
    ```text
    ##################################################################################
-   # CSM version:  1.2
+   # CSM version:  1.3
    # CANU version: 1.3.2
    ##################################################################################
    ```
 
-   - Output like the above text means that the switches have a CANU-generated configuration for CSM 1.2 in place. In this case, follow the steps in
-     [Management Network 1.0 (`1.2 Preconfig`) to 1.2](../../operations/network/management_network/1.0_to_1.2_upgrade.md).
-   - If the banner does NOT contain text like the above, then contact support in order to get the `1.2 Preconfig` applied to the system.
+   - Output like the above text means that the switches have a CANU-generated configuration for CSM 1.3 in place. In this case, follow the steps in
+     [Management Network 1.2 (`1.3 Preconfig`) to 1.3](../../operations/network/management_network/1.0_to_1.3_upgrade.md).
```

Comment:
Is there going to be a 1.3 Preconfig? I didn't think we were making the same big network changes moving to that release.